app/actions/index.ts

```diff
@@ -69,5 +72,29 @@ export function initialiseDefaults(
   currentUrl: string,
   interceptStatus: string
 ) {
-  return { type: INITIALISE_DEFAULTS, payload: { currentTab, currentUrl, interceptStatus } };
+  return {
+    type: INITIALISE_DEFAULTS,
+    payload: { currentTab, currentUrl, interceptStatus }
```

Comment:
Since we already have `initialState`, why we need to send the payload?

Reply:
Although `initialState` is defined in the reducer, we dont have any data about the `currentTabId`, `currentTabUrl`. Therefore, I dispatch this action as soon as the `popup` is opened so that we can store information based on `tabId`. Better to rename this as `currentTabId`, `currentTabUrl`.

I also dispatch `interceptStatus` with empty string so that after each time the popup is opened, the `Success` or `Error` message gets erased.